pandas/io/parsers.py

```diff
@@ -60,11 +60,18 @@
 
 Parameters
 ----------
-filepath_or_buffer : str, pathlib.Path, py._path.local.LocalPath or any \
-object with a read() method (such as a file handle or StringIO)
-    The string could be a URL. Valid URL schemes include http, ftp, s3, and
-    file. For file URLs, a host is expected. For instance, a local file could
-    be file://localhost/path/to/table.csv
+filepath_or_buffer : str, path object, or file object
```

Comment:
maybe "file-like" instead of "file" ? as eg StringIO is not really a actual file

Reply:
OK, if they use it like that, that's fine